site/tutorials/keptn-litmus-018.md

```diff
@@ -0,0 +1,378 @@
+
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite. 
+
+* To install the Prometheus and Alert Manager, execute:
+<!-- command -->
+```
+kubectl create ns monitoring
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm install prometheus prometheus-community/prometheus --namespace monitoring
+```
+
+### Execute the following steps to install prometheus-service
+
+* Download the Keptn's Prometheus service manifest
+
+```
+helm upgrade --install -n keptn prometheus-service \
+https://github.com/keptn-contrib/prometheus-service/releases/download/0.9.1/prometheus-service-0.9.1.tgz \
+--reuse-values \
+--set prometheus.namespace="monitoring" \
+--set prometheus.endpoint="http://prometheus-server.monitoring.svc.cluster.local:80" \
+--set prometheus.namespace_am="monitoring"
+```
+
+### Optional: Verify Prometheus setup in your cluster
+
+* To verify that the Prometheus scrape jobs are correctly set up, you can access Prometheus by enabling port-forwarding for the prometheus-service:
+<!-- command -->
+```
+kubectl port-forward svc/prometheus-server 8080:80 -n monitoring
+```
+
+
+## Setup Litmus integration
+Duration: 1:00
+
+Similar to the Prometheus integration, we are now adding the Litmus integration. This integration will be responsible to trigger the experiments with Litmus and listens for `sh.keptn.event.test.triggered` events that are sent from Keptn.
+
+This can be done via the following command.
+
+```
+kubectl apply -f ../deploy/service.yaml
+```
+
+We now have all the integrations installed and connected to the Keptn control plane. Let's move on with setup up a project!
+
+## Create project
+Duration: 1:00
+
+A project in Keptn is the logical unit that can hold multiple (micro)services. Therefore, it is the starting point for each Keptn installation.
+We have already cloned the demo resources from Github, so we can go ahead and create the project.
+
+
+**Recommended:** Create a new project with Git upstream:
+
+To configure a Git upstream for this tutorial, the Git user (`--git-user`), an access token (`--git-token`), and the remote URL (`--git-remote-url`) are required. If a requirement is not met, go to [the Keptn documentation](https://keptn.sh/docs/0.9.0/manage/git_upstream/) where instructions for GitHub, GitLab, and Bitbucket are provided.
+
+Let's define the variables before running the command:
+
+<!-- bash keptn create project litmus --shipyard=./shipyard.yaml -->
+
+```
+GIT_USER=gitusername
+GIT_TOKEN=gittoken
+GIT_REMOTE_URL=remoteurl
+```
+
+Now let's create the project using the `keptn create project` command.
+
+```
+keptn create project litmus --shipyard=./shipyard.yaml --git-user=$GIT_USER --git-token=$GIT_TOKEN --git-remote-url=$GIT_REMOTE_URL
+```
+
+For creating the project, the tutorial relies on a `shipyard.yaml` file as shown below:
```

Comment:
I don't know if you want to do xrefs here but the shipyard reference file is at https://keptn.sh/docs/0.18.x/reference/files/shipyard/ .  That saves you from having to talk about what shipyard is, but also gives a lifeline to a user who isn't familiar with the shipyard.